atlp-rwanda/trojans-ec-fe
# #184760063 Payment Via Stripe

### What does this PR do?

This PR adds the functionality of payment via Stripe on our app.

### Description of Task to be completed?

Integrating Stripe API allows for a secure and efficient way for buyers to make payments and complete their transactions.

### How should this be manually tested?

login as a buyer and add a product to the cart, then proceed to the cart page and click proceed checkout, if checkout succeeds you will see the link proceed payment and the click it, you will be follow the link to stripe and enter payment information, after if the payment succeeds, you will be redirected to the page that confirms your payment with the summary of what you have paid for.
[follow this link for test preview](https://trojans-ec-fe-git-ft-payment-184760063-atlp-trojan.vercel.app/)

### Any background context you want to provide?

N/A

### What are the relevant pivotal tracker stories?

[Provide a relevant link to the pivotal tracker.](https://www.pivotaltracker.com/story/show/184760063)

### Screenshots (if appropriate)
#Before payment
![image](https://github.com/atlp-rwanda/trojans-ec-fe/assets/50501079/9c7c037f-efab-4fda-b352-892e85a63e02)

#after payment
![image](https://github.com/atlp-rwanda/trojans-ec-fe/assets/50501079/69e192e6-3d69-44da-996d-b6653feb3db0)

### Questions:

# Type of change

Please check the options that are relevant.


# Checklist:
